core/benches/banking_stage.rs

```diff
@@ -125,21 +128,27 @@ fn bench_banking_stage_multi_accounts(bencher: &mut Bencher) {
         );
         poh_recorder.lock().unwrap().set_bank(&bank);
 
-        let mut id = genesis_block.hash();
-        for _ in 0..(MAX_RECENT_BLOCKHASHES * DEFAULT_TICKS_PER_SLOT as usize) {
-            id = hash(&id.as_ref());
-            bank.register_tick(&id);
-        }
-
         let half_len = verified.len() / 2;
         let mut start = 0;
+
+        // So that the signal_receiver does not go out of scope after the closure
```

Comment:
This comment seems out of left field. Also, it's written as a sentence, but it's incomplete.